mlflow/paddle/__init__.py

```diff
@@ -393,3 +423,9 @@ def predict(self, data):
 
         predicted = self.pd_model(inp_data)
         return pd.DataFrame(predicted.numpy())
+
+
+def _check_if_model_supports_retrain(path):
+    print(path)
+    file_list = os.listdir(path)
+    return any(".pdparams" in file for file in file_list)
```

Comment:
```suggestion
def _check_if_model_supports_retrain(path):
    file_list = os.listdir(path)
    return any(".pdparams" in file for file in file_list)
```

Now I think we should rename this function and remove `print`.

Reply:
How about ```_check_if_model_saved_in_pdparams_format```

Reply:
What about `_contains_pdparams`? It matches what this function does.

Reply:
Year, cool!